site/docs/contribute/environment.md

```diff
@@ -36,11 +36,12 @@ This installs additional dependencies to `hack/tools/bin`. The `make` targets (e
 The project is predominantly written in Go so we recommend [Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-vscode.Go) for its good Go support. Alternatively you can select from [Editors](https://golang.org/doc/editors.html)
 
 ## Docker hub rate limits
+
 As docker hub introduced rate limits on docker image downloads this may effect development using the local kind setup.
 One option to fix the limit is to use a docker hub login for downloading the images. The environment will run
 a docker registry as a proxy for all public images. This registry runs in a docker container next to the kind clusters. 
 
 ```shell
-export DOCKERHUB_USERNAME='your docker hub username'
-export DOCKERHUB_PASSWORD='your password'
+export DOCKER_USERNAME='your github username'
```

Comment:
This section is about docker hub rate limits (for base images taken from dockerhub). The original text is correct. If there is something that doesn't work for you please mention it explicitly in an issue because I don't get it :-)

Reply:
@roee88 make-notebook-tests and make-integration-tests fails if DOCKER_USERNAME and DOCKER_PASSWORD are not defined.

Reply:
The rules don't fail. Makefile sets these vars explicitly if not defined, for example:
```
run-notebook-tests: export DOCKER_HOSTNAME?=localhost:5000
run-notebook-tests: export DOCKER_NAMESPACE?=fybrik-system
```

Reply:
Doesn't work for me on my mac.  Perhaps it's a local issue.  I will revert.